src/Components/List/List.jsx

```diff
@@ -118,8 +118,42 @@ export default function List({ items, token }) {
         item.name.toLowerCase().includes(searchItem.toLocaleLowerCase()),
       );
 
-  // Delete modal confirmation
-  const [itemToDelete, setItemToDelete] = useState('');
+  // Modal to delete item
+  const [modalIsOpen, setModalIsOpen] = useState(false);
+  const [modalMessage, setModalMessage] = useState('');
+  const [confirmFunction, setConfirmFunction] = useState(null);
+  const [modalLabel, setModalLabel] = useState('');
+
+  const closeModal = () => {
+    setModalIsOpen(false);
+  };
+
+  function showAlert(message) {
+    setModalMessage(message);
+    setModalLabel('Alert');
```

Comment:
We tested out the modal with Voiceover and it works really well! We did notice though that the screen reader just alerts the user to let them know that the window dialog is an alert, but doesn't specify what the alert message is. Maybe you could pass `message` into the `setModalLabel` function in addition to "Alert" to make it more specific?

Reply:
We just noticed that the `aria-label` for the alerts that get displayed after a user confirms deletion weren't being read at all. This might be because we're just changing the alert message/label of the existing modal and not actually creating a new one.

Any thoughts @mxmason ? Would this be a case for ARIA live region? I know that you mentioned in the past that it is a little tricky to make it functional in React.